### PR TITLE
New package: thermald-1.4.2

### PR DESCRIPTION
**Pull request by thypon**

### CODE REVIEW

File: srcpkgs/thermald/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'thermald'
+pkgname=thermald
```
Comment (Gottox) [inline srcpkgs/thermald/template]: why not call it thermal_daemon like the package is actually named?

Comment (thypon) [inline srcpkgs/thermald/template]: Bacause the (only) binary and the service are called thermald: plus the name is thermald in ubuntu/debian/arch/fedora.

**Comment by xtraeme** [comment]
Looks ok :+1:

File: srcpkgs/thermald/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+version=1.4.2
+revision=1
+_pkgname=thermal_daemon
+wrksrc="${_pkgname}-${version}"
+only_for_archs="i686 i686-musl x86_64 x86_64-musl"
```
Comment (beefcurtains) [inline srcpkgs/thermald/template]: Why this? Does Linux not support thermal devices on ARM?

Comment (thypon) [inline srcpkgs/thermald/template]: 01.org it's intel. It's an intel specific piece of software; uses intel technologies to achieve better thermal management.

**Pull request closed without merging**